docker/Dockerfile.qdrant-loader

```
@@ -0,0 +1,42 @@
+    && rm -rf /var/lib/apt/lists/*
+# git must be available at runtime
+# libgl1 + libglib2.0-0: runtime shared libs for opencv-python, which docling pulls
+# (via rapidocr / docling-ibm-models) for image preprocessing in the PDF layout
+# pipeline. python:slim ships neither, so cv2 import fails with "libGL.so.1: cannot
+# open shared object file" during PDF conversion.
+
+USER appuser
+
+ARG QDRANT_LOADER_VERSION=latest
+
+RUN if [ "$QDRANT_LOADER_VERSION" = "latest" ]; then \
```

Hadolint (2.14.0)

[warning] Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>` or `pip install --requirement <requirements file>`

(DL3013)